system/jlib/jevent.cpp

```diff
@@ -1757,6 +1821,513 @@ IEventVisitor* createDumpCSVEventVisitor(IBufferedSerialOutputStream& out)
     return new CDumpCSVEventVisitor(out);
 }
 
+class CEventFilter : public CInterfaceOf<IEventFilter>
+{
+protected:
+    class StringMatchHelper
+    {
+    private:
+        using Patterns = std::set<std::string>;
+        Patterns patterns;
+
+    public:
+        bool matches(const char* value) const
+        {
+            std::string tmp(value);
+            for (const std::string& pattern : patterns)
+            {
+                if (tmp.compare(pattern) == 0)
+                    return true;
+                if (isWildString(pattern.c_str()))
```

Comment:
This will match "abc.*".  I am not sure wildcards add anything - regex should be sufficient.